build.sbt

```diff
@@ -117,6 +117,7 @@ val customMergeStrategy: String => MergeStrategy = {
     }
   case PathList("javax", "annotation", _@_*) => MergeStrategy.first
   case "NOTICE" => MergeStrategy.discard
+  case "LICENSE" => MergeStrategy.first
```

Comment:
with the new sendgrid-java jar, `sbt assembly` failed to merge due to a LICENSE file; this fixes it